service/notesservice.php

```diff
@@ -132,7 +132,7 @@ public function create ($userId) {
         // check new note exists already and we need to number it
         // pass -1 because no file has id -1 and that will ensure
         // to only return filenames that dont yet exist
-        $path = $this->generateFileName($folder, $title, "txt", -1);
+        $path = $this->generateFileName($folder, $title, $this->settings->get('fileSuffix'), -1);
```

Comment:
why the change for `fileSuffix` instead of `extension` in all files?

Reply:
Is your comment about the naming? I chose `fileSuffix` because the value includes the dot. I wanted to have the dot included, because it's more generic and I think that displaying `.md` to the user is more helpful than displaying `md`.